pandas/core/generic.py

```diff
@@ -3563,6 +3563,42 @@ def head(self, n=5):
         -------
         obj_head : type of caller
             The first n rows of the caller object.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({
+        ...     'col1' : ['A', 'A', 'B', np.nan, 'D', 'C', 'E', 'E', 'F'],
+        ...     'col2' : [2, 1, 9, 8, 7, 4, 0, 2, 2],
+        ...     'col3': [0, 1, 9, 4, 2, 3, 5, 7, 3],
+        ... })
+        >>> df
+            col1 col2 col3
+        0   A    2    0
+        1   A    1    1
+        2   B    9    9
+        3   NaN  8    4
```

Comment:
Why are you adding a nan value?
This is creating a mixed type column. (floats and strings) is this intentional?

Reply:
NaN is used for marking missing values in string (object) dtype columns too, so this is OK.

Reply:
just copied from an example above.